### PR TITLE
Parameters api fixes

### PR DESCRIPTION
**Pull request by tstrachota**
- fixed apipie docs
- rabl definition changed to match conventions

### CODE REVIEW

**Comment by isratrade** [comment]
looks good.  thanks

File: app/views/api/v2/parameters/index.json.rabl [hunk 1]
```diff
@@ -1,3 +1,3 @@
-collection @parameters => :parameters
+collection @parameters, :object_root => :parameter
```
Comment (isratrade) [inline app/views/api/v2/parameters/index.json.rabl:1]: Are you sure this is :parameter and not :parameters?

Comment (tstrachota) [inline app/views/api/v2/parameters/index.json.rabl:1]: Yes, this is object root not collection root. We don't use collection roots anywhere else do we?

Comment (isratrade) [inline app/views/api/v2/parameters/index.json.rabl:1]: @tstrachota, looking at the rabl documentation, it looks like what I had "@parameters => :parameters" creates :root => :parameters and :object_root => :parameter.  Is there another reason for your change.

Comment (tstrachota) [inline app/views/api/v2/parameters/index.json.rabl:1]: No other reason. I just wanted to get rid of the root "parameters" while keeping the object root the same. We don't use root for collections in other controllers so I wanted to make this one consistent as well.

Comment (isratrade) [inline app/views/api/v2/parameters/index.json.rabl:1]: Ok, sounds good. I understand now.

**Comment by domcleal** [comment]
@tstrachota could you create a redmine ticket for this please?  Set the status to ready for testing, link to the PR and target version of 1.3.  Thanks!

**Comment by tstrachota** [comment]
Fixed http://projects.theforeman.org/issues/2803

**Comment by domcleal** [comment]
Thanks @tstrachota, merged as 9903b42.  I've updated the commit message so it's on one line and contains the issue number.

**Pull request closed without merging**